components/de_amp/browser/de_amp_url_loader.h

```diff
@@ -45,6 +45,7 @@ class DeAmpURLLoader : public body_sniffer::BodySnifferURLLoader {
 
   void OnBodyReadable(MojoResult) override;
   void OnBodyWritable(MojoResult) override;
+  bool found_amp_ = false;
```

Comment:
normally, methods and variables are different "groups" inside the class definition, see https://google.github.io/styleguide/cppguide.html#Declaration_Order